pylmod/tests/test_gradebook.py

```diff
@@ -805,13 +807,195 @@ def test_spreadsheet2gradebook_multi(self):
             ]
         )
 
+    @data(
+        (50, 0, True, 50),  # happy case
+        ('50', '0', True, 50),  # happy case with strings
+        ('', '0', True, 1),  # max_points is an invalid number
+        (None, '0', True, 1),  # max_points is missing
+        ('50', '1', True, 1),  # normalize is true
+        ('50', '', True, 1),  # normalize is an invalid number
+        ('50', None, True, 1),  # normalize is missing
+        ('50', '0', False, 1),  # use_max_points_column is False
+    )
+    @unpack
+    @httpretty.activate
+    def test_use_max_points_column(
+            self, max_points, normalize, use_max_points_column,
+            expected_max_points):
+        """
+        If use_max_points_column is True and normalize? is False,
+        use the max_points value.
+        """
+        new_assignment_id = 3
```

Comment:
Yes, it's just an unused assignment id. 1 and 2 are already taken in the setup method